connector-packager/tests/test_resources/instanceurl/valid-connector/oauthConfig.xml

```diff
@@ -0,0 +1,61 @@
+<pluginOAuthConfig>
+    <!-- Copyright (C) 2017-2019 Dremio Corporation. This file is confidential and private property. -->

```

Comment:
This file is still has references to Dremio. We have a valid oauth test connector [here](https://github.com/tableau/connector-plugin-sdk/tree/master/connector-packager/tests/test_resources/oauth_connector), use that as a base. You can probably also re-use that one outright.

Reply:
Had forgotten to add the updated files which I added later. Will try also try with the generic OAuth connector already present in the repo and replace it with that if it works.